jimp/index.d.ts

```diff
@@ -0,0 +1,163 @@
+// Type definitions for jimp 0.2
+// Project: https://github.com/oliver-moran/jimp#readme
+// Definitions by: Jack Works <https://github.com/Jack-Works>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+
+declare namespace jimp {
+	type Callback<T> = (this: JimpImage, err: Error, data: T) => void;
+	type ResizeMode = string;
+	type PresetFont = string;
+	type FileMINE = string;
+	type AlignMode = number;
+	type FilterType = number;
+	type Font = any | PresetFont;
+
+
+	class JimpImage {
+		constructor(width: number, height: number, callback?: Callback<JimpImage>);
+		constructor(width: number, height: number, initialColor: number, callback?: Callback<JimpImage>);
+		/* Get/set meta data */;
+		hash(n?: number): string;
+
+		getExtension(): string;
+		getMIME(): FileMINE;
+
+		getPixelColor(x: number, y: number): number;
+		setPixelColor(hex: number, x: number, y: number): void;
+
+		/* Save */;
+		write(path: string, callback?: Callback<void>): void;
+		getBase64(mine: FileMINE, callback?: Callback<string>): void;
+		getBuffer(mine: FileMINE, callback?: Callback<Buffer>): void;
+
+		/* Parameter */;
+		quality(n: number): JimpImage;
+		rgba(is: boolean): JimpImage;
+		filterType(type: FilterType): JimpImage;
+		deflateLevel(level: number): JimpImage;
+
+		/* Operation */;
+		clone(): JimpImage;
+		print(font: Font, x: number, y: number, text: string, width?: number): JimpImage;
+
+		/* Adv */;
+		color(params: Array<{apply: 'lighten' | 'brighten' | 'darken' |
```

Comment:
This formatting looks wierd. Can you put the parameters in a new line if it is too long?